### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/imexam/commit/a5f5aae92095ca63de93045ba1789d75523ea90f#commitcomment-62588515

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/imexam/commit/19f09d4cecc73cafbebf7a30c5edf2540429d743

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

I intend to annotate this PR shortly. I'm happy to drop changes to files, drop specific terms, or split things as requested, although this may take some time.

### CODE REVIEW

File: docs/imexam/imexam_command.rst [hunk 1]
```diff
@@ -125,7 +125,7 @@ This will use Ginga (instead of the default DS9) for the viewer::
     :alt: Data used for imexam command examples
 
 
-Circular Apterture Photometry
+Circular Aperture Photometry
 -----------------------------
```
Comment (jsoref) [inline docs/imexam/imexam_command.rst:129]: Oops, need to trim `-`s

File: docs/imexam/walkthrough.rst [hunk 2]
```diff
@@ -81,7 +81,7 @@ displays the same image to each one::
 
 
 It's also possible to load a FITS image object that you already have opened in your
-python session, if no extension is given, then the first IMAGE exension that is found
+python session, if no extension is given, then the first IMAGE extension that is found
```
Comment (jsoref) [inline docs/imexam/walkthrough.rst:84]: I know there's an XTENSION thing nearby, but I think this use can use the normal spelling.

File: CHANGES.rst [hunk 3]
```diff
@@ -68,7 +68,7 @@ version 0.7.0 (2017-01-19)
 - fixed a text error in the display_help() so that now the correct version loads the documentation
 - Windows users can now install from source. The setup will ignore the cython and xpa necessary to build the DS9 interaction, and users will only be able to use the Ginga HTML5 window, they can also use the Imexamine() functions without any graphical interface.
 - Documentation updates, mostly specific information for Windows users
-- Added python 3.6 to the test matrix as well as apveyor for the windows build
+- Added python 3.6 to the test matrix as well as appveyor for the windows build
```
[the remark this answers is not in the document]
Comment (larrybradley) [reply CHANGES.rst:71]: Yes, this should be AppVeyor

File: docs/imexam/imexam_command.rst (hunk 1 above)
Comment (larrybradley) [reply docs/imexam/imexam_command.rst:129]: Can you trim in this PR?

**Review by larrybradley — APPROVED** [review]
@jsoref - Many thanks!

**Comment by jsoref** [comment]
@larrybradley: The cleanest way to fix `AppVeyor` was to do it for 2 files consistently and at the same time also adjust `Travis` and `Windows` and `Python` / `IPython`. I can drop those if they're problematic.

**Review by larrybradley — APPROVED** [review]
Thanks!

**Pull request merged**